Core/cafObjectMethod.h

```diff
@@ -79,7 +79,13 @@ class ObjectMethod : public Object
     CAFFA_HEADER_INIT;
 
 public:
-    ObjectMethod( caffa::not_null<ObjectHandle*> self );
+    enum class Type
+    {
+        NON_CONST = 0,
+        CONST
+    };
```

Comment:
Maybe READ_WRITE and READ_ONLY is better here? OBSERVING vs CHANGING?

Reply:
godt poeng. Sleit litt med å finne gode navn!